code/data_preprocessing/genotype/PCA.ipynb

```diff
@@ -843,9 +843,9 @@
     "    if (nrow(f$loadings) != length(ref)) stop(\"Input number of variants should be the same as used in the previous PCA model\")\n",
     "    overlapped_variants <- match(names(ref), rownames(f$loadings))\n",
     "    # Project related samples\n",
-    "    fpro <- project(frel, loadings=f$loadings[overlapped_variants,], orig_mean=f$center[overlapped_variants], orig_sd=f$scale[overlapped_variants], ref_allele=ref)\n",
-    "    pca <- fpro$projection\n",
     "    k = ${k}\n",
+    "    fpro <- project(frel, loadings=f$loadings[overlapped_variants,1:k], orig_mean=f$center[overlapped_variants], orig_sd=f$scale[overlapped_variants], ref_allele=ref)\n",
```

Comment:
I dont understand why this is necessary ... `f$loadings` should have `k` columns. If it's not `k` columns something is wrong here. We cannot just truncate it.